<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParoValMVPClass</class>
 <widget class="QMainWindow" name="ParoValMVPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParOVal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="startPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Calculate Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closePushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Grid Settings</string>
    </property>
    <widget class="QCheckBox" name="showGridCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Grid</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gridCellSizeLineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid Cell Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph Settings</string>
    </property>
    <widget class="QCheckBox" name="showGraphCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Graph</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showNodeTextCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Node Text</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showNodeCostsCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Node Costs</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>531</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Path Finding Settings</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Obstacle Cost Factor</string>
     </property>
    </widget>
    <widget class="QSlider" name="obstacleCostHorizontalSlider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="pageStep">
      <number>100000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="obstacleCostLineEdit">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth Rate Apha</string>
     </property>
    </widget>
    <widget class="QSlider" name="smoothAlphaHorizontalSlider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="smoothAlphaLineEdit">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>150</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth Rate Beta</string>
     </property>
    </widget>
    <widget class="QSlider" name="smoothBetaHorizontalSlider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="smoothBetaLineEdit">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>190</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="randomPointsCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Random Start / End Points</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Point</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="startPointLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>End Point</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endPointLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="visualizePushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize Optimization</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="inputPushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Local</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputImageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>81</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Local File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenScenario DB</string>
    </property>
   </widget>
   <widget class="QComboBox" name="openScenarioComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="getPushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get from Cloud</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ParoValMVP.qrc"/>
 </resources>
 <connections/>
</ui>
